src/CircleGraph/CircleViewer.ts

```diff
@@ -123,9 +123,7 @@ export class CircleViewerProvider implements
   }
 
   // CustomReadonlyEditorProvider implements
-  async openCustomDocument(
-      uri: vscode.Uri, openContext: {backupId?: string},
-      _token: vscode.CancellationToken): Promise<CircleViewerDocument> {
+  async openCustomDocument(uri: vscode.Uri): Promise<CircleViewerDocument> {
```

Comment:
```suggestion
  async openCustomDocument(uri: vscode.Uri, _openContext: {backupId?: string},
      _token: vscode.CancellationToken): Promise<CircleViewerDocument> {
```
Like this.

(seems you already put the underscore for _token..?)